app/src/main/java/githubuser/model/GithubUsers.java

```diff
@@ -0,0 +1,25 @@
+package githubuser.model;
+
+import com.google.gson.annotations.SerializedName;
+
+public class GithubUsers {
+
+    @SerializedName("userName")
+    private String userName;
+
+    @SerializedName("userOrg")
```

Comment:
same applies here
 https://developer.github.com/v3/search/#search-users

this will help u to know wat am saying